packages/tests/server/react/invalidateQueries.test.tsx

```diff
@@ -186,7 +186,7 @@ describe('invalidateQueries()', () => {
           <button
             data-testid="invalidate-4-predicate"
             onClick={() => {
-              utils.invalidate({
+              utils.invalidate(undefined, {
```

Comment:
Isn't this changing the behaviour? 🤔 

Looks like a breaking change?

Reply:
I mean yea theoretically although it wasnt working before so its more of a fix?

Reply:
Mentioned this here: https://discord.com/channels/867764511159091230/1055437438266392646/1055450208424706099

We always take the first arg as input no matter what the types said so any filters applied would actually end up being supressed